.github/workflows/build.yml

```diff
@@ -31,8 +31,7 @@ jobs:
           pip install coverage wheel pytest
           pip install -e ".[postgresql]"
       - name: Run the tests
-        run: |
-          pytest
+        run: make test
```

Comment:
This is personal preference, but I’m not a huge fan of running tests in CI within Docker containers if the environment (like in this case) is reasonable simple. The tests could probably run in ~10 seconds, but now it takes 2+ mins because the Docker image has to be built etc.

Reply:
We are running some tests against a postgres database which we need to spin up a Docker image for. Does this feel like an acceptable compromise, in this case?

Reply:
In general I agree with Till. But we need the db container and since we're mounting the source code as a volume into the container this is an acceptable drawback for development (i.e. allows quick iterations) and it follows the same patterns we have in other repos (not that that's an excuse).